### ISSUE TITLE
`svelte-ignore` comments not honored in expression positions (object properties; stacked comments before call arguments)

### ISSUE THREAD

**Issue by MathiasWP** [harshmandan/svelte-check-native#37]
## Summary

`// svelte-ignore <code>` comments in the `<script>` block are honored at statement level, but **dropped in some expression positions**, producing false warnings that `svelte-check` correctly suppresses. On our ~2,100-file SvelteKit app this produced 31 false warnings (all at sites explicitly suppressed with `svelte-ignore`).

## Environment

- svelte-check-native **1.0.2** (darwin-arm64), `TSGO_BIN` → `@typescript/native-preview` 7.0.0-dev.20260707.2
- Compared against `svelte-check` **4.7.3** with `svelte` 5.56.5 / TypeScript 6.0.3

## Variant matrix (all verified)

| Position of `// svelte-ignore state_referenced_locally` | svelte-check 4.7.3 | svelte-check-native 1.0.2 |
|---|---|---|
| Statement level, single comment | suppressed | suppressed ✅ |
| Statement level, stacked under `// eslint-disable-next-line …` | suppressed | suppressed ✅ |
| Before a call argument, single comment | suppressed | suppressed ✅ |
| Before a call argument, stacked under another comment | suppressed | **warns** ❌ |
| Before an object literal property, single comment | suppressed | **warns** ❌ |
| Before an object literal property, stacked | suppressed | **warns** ❌ |

## Repro A — object property (single comment is enough)

```svelte
<script lang="ts">
	let count = $state(0);

	const sections = [
		{
			value: 'a',
			// svelte-ignore state_referenced_locally
			open: count > 0
		}
	];
</script>

<button onclick={() => count++}>{sections[0].open} / {count}</button>
```

```
WARNING "src/IgnoreOnObjectPropSingle.svelte" 8:10 "This reference only captures the initial value of `count`. …"
```

## Repro B — stacked comments before a call argument

```svelte
<script lang="ts">
	let { projectId, criterion }: { projectId: string; criterion: string } = $props();

	class Persisted<T> {
		constructor(
			public key: string,
			public value: T
		) {}
	}

	const thing = new Persisted<{ title: string }>(
		// eslint-disable-next-line svelte/no-unused-svelte-ignore
		// svelte-ignore state_referenced_locally
		`key-${projectId}-${criterion}`,
		{ title: '' }
	);
</script>

<p>{thing.key}</p>
```

```
WARNING "src/IgnoreInNewExpr.svelte" 14:10 "This reference only captures the initial value of `projectId`. …"
WARNING "src/IgnoreInNewExpr.svelte" 14:23 "This reference only captures the initial value of `criterion`. …"
```

## Expected

`svelte-check` 4.7.3 reports zero warnings for both files — the svelte compiler associates leading `svelte-ignore` comments with the following node in these positions.

Related: #36 (found in the same evaluation). Happy to test fixes.

**Comment by harshmandan** [comment]
Fixed in release [v1.1.0](https://github.com/harshmandan/svelte-check-native/releases/tag/v1.1.0)
